start.sh

```diff
@@ -3,6 +3,8 @@
 # Chown volumes so that nginx can read/write
 chown www-data:www-data /usr/share/nginx/html/images
 chown www-data:www-data /usr/share/nginx/html/LocalSettings.php
+chown -R www-data:www-data /tmp/extensions/
+cp -r /tmp/extensions/* /usr/share/nginx/html/extensions/
```

Comment:
You can just mount it directly to that folder and avoid the `cp`. For example `-v <path to extensions>:/usr/share/nginx/html/extensions`